packages/database/lib/migrations/20250310134848_connection_add_meta_fields.cjs

```diff
@@ -0,0 +1,23 @@
+/**
+ * @param {import('knex').Knex} knex
+ */
+exports.up = async function (knex) {
+    await knex.raw(`ALTER TABLE "_nango_connections"
+ADD COLUMN "credentials_expires_at" timestamptz,
+ADD COLUMN "last_refresh_success" timestamptz,
+ADD COLUMN "last_refresh_failure" timestamptz,
+ADD COLUMN "last_refresh_tries" int2,
+ADD COLUMN "refreshable" bool`);
+};
+
+/**
+ * @param {import('knex').Knex} knex
+ */
+exports.down = async function (knex) {
+    await knex.raw(`ALTER TABLE "_nango_connections"
+DROP COLUMN "credentials_expires_at",
+DROP COLUMN "last_refresh_success",
+DROP COLUMN "last_refresh_failure",
+DROP COLUMN "last_refresh_tries",
+DROP COLUMN "refreshable"`);
```

Comment:
I personally have gave up on the `down`. :D

Reply:
Yeah I'm trying to be a good citizen but it's useless